Source/Urho3D/Network/HttpRequest.cpp

```diff
@@ -109,6 +112,14 @@ void HttpRequest::ThreadFunction()
             headersStr += header + "\r\n";
     }
 
+#ifdef URHO3D_SSL
+    if (!sSSLInitialized)
+    {
+        mg_init_library(MG_FEATURES_TLS);
+        sSSLInitialized = true;

```

Comment:
variable name must be updated

Reply:
But now that I think about it, it would be probably better to just put:
~~~cpp
#ifdef URHO3D_SSL
    static sslInitialized = false;
    if (!sslInitialized)
    {
        mg_init_library(MG_FEATURES_TLS);
        sslInitialized = true;
    }
#endif
~~~
in the Constructor.

Reply:
Not a bad idea.